resources/themes/cura-light/icons/default/Certified.svg

```diff
@@ -0,0 +1,3 @@
.32281 15.1035 3.52741 15.4404 3.695C15.7969 3.81446 16.1656 3.89405 16.5396 3.9323C17.4722 3.97169 18.3654 4.31907 19.0796 4.9201C19.6646 5.61964 20.0048 6.49125 20.0484 7.4021C20.0908 7.7964 20.1769 8.18476 20.3052 8.56C20.4798 8.91626 20.6935 9.25196 20.9424 9.5609C21.5557 10.2357 21.9315 11.0926 22.0124 12.0009C21.9325 12.9307 21.5466 13.8078 20.9152 14.495C20.6774 14.7866 20.4728 15.1039 20.3052 15.4408C20.1857 15.7973 20.1061 16.166 20.0679 16.54C20.0285 17.4726 19.6811 18.3658 19.0801 19.08C18.3807 19.6653 17.509 20.0055 16.5981 20.0488C16.2038 20.0912 15.8155 20.1775 15.4404 20.3061C15.0841 20.4805 14.7484 20.694 14.4395 20.9428C13.7649 21.5557 12.9084 21.9313 12.0005 22.0122ZM12 3.9878C11.5315 4.1067 11.1003 4.34102 10.7456 4.6694C10.309 5.01634 9.83223 5.30948 9.32559 5.5425C8.80239 5.73609 8.25776 5.86593 7.70349 5.9292C7.22072 5.94798 6.7503 6.08713 6.33499 6.334C6.07267 6.76174 5.9264 7.25048 5.91059 7.752C5.8537 8.28949 5.73005 8.81779 5.54249 9.3247C5.31656 9.81621 5.03019 10.2776 4.68999 10.6982C4.34665 11.0638 4.10465 11.5127 3.98789 12.0005C4.10683 12.4686 4.34097 12.8994 4.66899 13.2539C5.01645 13.6907 5.30978 14.168 5.54259 14.6753C5.73618 15.1983 5.86602 15.7428 5.92929 16.2969C5.94811 16.7796 6.08744 17.2499 6.33459 17.665C6.76211 17.9274 7.25071 18.0737 7.75209 18.0894C8.28958 18.1463 8.81789 18.2699 9.32479 18.4575C9.81631 18.6835 10.2777 18.9699 10.6983 19.3101C11.0639 19.6534 11.5128 19.8954 12.0006 20.0122C12.4691 19.8933 12.9003 19.659 13.255 19.3306C13.6916 18.9837 14.1684 18.6905 14.675 18.4575C15.1982 18.2639 15.7428 18.1341 16.2971 18.0708C16.7797 18.052 17.25 17.9128 17.6652 17.666C17.9275 17.2383 18.0738 16.7495 18.0896 16.248C18.1465 15.7105 18.2701 15.1822 18.4577 14.6753C18.6837 14.1838 18.9701 13.7224 19.3103 13.3018C19.6535 12.9363 19.8954 12.4876 20.0122 12C19.8933 11.5319 19.6591 11.1011 19.3311 10.7466C18.9836 10.3098 18.6903 9.83249 18.4575 9.3252C18.2639 8.80217 18.1341 8.25771 18.0708 7.7036C18.052 7.22089 17.9126 6.75057 17.6655 6.3355C17.238 6.0731 16.7494 5.92681 16.248 5.9111C15.7105 5.85421 15.1822 5.73055 14.6753 5.543C14.1838 5.31692 13.7224 5.03037 13.3018 4.69C12.9363 4.3467 12.4876 4.10467 12 3.9878ZM15.5361 15.5361L14.1221 14.1211C13.7026 14.5407 13.168 14.8265 12.5861 14.9423C12.0041 15.0581 11.4009 14.9987 10.8526 14.7717C10.3044 14.5447 9.83586 14.1601 9.50619 13.6668C9.17652 13.1734 9.00055 12.5934 9.00055 12C9.00055 11.4066 9.17652 10.8266 9.50619 10.3332C9.83586 9.83986 10.3044 9.45535 10.8526 9.2283C11.4009 9.00126 12.0041 8.94188 12.5861 9.05769C13.168 9.1735 13.7026 9.45928 14.1221 9.8789L15.5361 8.4639C14.8368 7.7648 13.9458 7.28875 12.976 7.09596C12.0061 6.90316 11.0009 7.00227 10.0873 7.38076C9.1738 7.75924 8.393 8.4001 7.84367 9.22232C7.29433 10.0445 7.00113 11.0112 7.00113 12C7.00113 12.9888 7.29433 13.9555 7.84367 14.7777C8.393 15.5999 9.1738 16.2408 10.0873 16.6192C11.0009 16.9977 12.0061 17.0968 12.976 16.904C13.9458 16.7112 14.8368 16.2352 15.5361 15.5361Z" fill="#000E1A"/>
```

Comment:
The fill colour is incorrect here.

Reply:
What should be the suggested fill color? This is the icon as provided by UX

Reply:
Icons should not contain colors in the first place. Colors should be applied with UM.RecolorImage.

(Yes, CheckBlueBG.svg is an abomination, in more than one way)